docs/en/configuration/sink-plugins/Redis.md

```diff
@@ -0,0 +1,57 @@
+# Sink plugin: Redis
+
+### Description
+
+Write Rows to a Redis.
+
+### Options
+
+| name | type | required | default value | engine |
+| --- | --- | --- | --- | --- |
+| [redis_host](#redis-host-string) | string | yes | "localhost" | all streaming |
```

Comment:
```
| [redis_host](#redis_host-string) | string | yes | "localhost" | all streaming |
```
You should write like this, otherwise the redirect would be invalid.